packages/11ty/_plugins/transforms/outputs/pdf/transform.js

```diff
@@ -87,6 +100,70 @@ module.exports = function(eleventyConfig, collections, content) {
     // 
   }
 
+  /**
+   * @function trimLeadingSeparator
+   * 
+   * Trims the publication URL path from @src attribs and style background-image URLs 
+   * 
+   **/
+  const trimLeadingSeparator = (document) => {
+
+    const urlPath = eleventyConfig.globalData.publication.pathname
+
+    /**
+     * Trim func, removes either the deploy path or just the leading slash 
+     */
+    const trimDeployPathComponentOrSlash = (srcAttr) => {
+      // - /foo/_assets/image.jpg -> _assets/image.jpg -- FIXME: How is it that the background-image doesn't need fixed?
+      // - /_assets/image.jpg -> _assets/image.jpg
+      // - Passes URLs with protocols
+      switch (true) {
+        case srcAttr.startsWith(urlPath):
+          return srcAttr.substr(urlPath.length)
+        case srcAttr.startsWith('/'):
+          return srcAttr.substr(1)
+        default:
+          return srcAttr
+      }
+    }
+
+    document.querySelectorAll('[src]').forEach((asset) => {
+      const src = asset.getAttribute('src')
+      asset.setAttribute('src', trimDeployPathComponentOrSlash(src))
+    })
+
+    document.querySelectorAll('[style*="background-image"]').forEach((element) => {
+      const backgroundImageUrl = element.style.backgroundImage.match(/[\(](.*)[\)]/)[1] || ''
+      element.style.backgroundImage = `url('${trimDeployPathComponentOrSlash(backgroundImageUrl)}')`
+    })
+
+  }
+
+  /**
+   * @function normalizeCoverPageData
+   * 
+   * @param {Object} pageData - page data object
+   * @param {Object} pdfConfig - configuration for the pdf
+   * 
+   * Returns {Object} data formatted for the layout at _layouts/pdf-cover-page.liquid
+   *   
+   **/
+  function normalizeCoverPageData(page,pdfConfig) { 
+
+    const { pagePDFCoverPageCitationStyle } = page.data
+
+    const id = `page-${page.data.key}`
+    const title = pageTitle(page.data)
+    const accessUrl = page.data.canonicalUrl
+    const contributors = JSON.stringify(page.data.pageContributors ?? '[]')     
+    const license = page.data.publication.license.name // FIXME: Need a license *text* ala https://www.getty.edu/publications/cultural-heritage-mass-atrocities/downloads/pages/CunoWeiss_CHMA_part-1-02-macgregor.pdf 
+    const copyright = page.data.publication.copyright
+    const pageCitation = (pagePDFCoverPageCitationStyle ?? quirePDFConfig?.pagePDF?.coverPageCitationStyle ) ? citation({context: 'page',page, type: pagePDFCoverPageCitationStyle ?? quirePDFConfig?.pagePDF?.coverPageCitationStyle }) : ''
+
+    return { id, title, accessUrl, contributors, license, copyright, citation: pageCitation }
```

Comment:
Format return object on multiple line and sort properties to improve readability;
```suggestion
    return { 
      accessUrl,
      citation: pageCitation, 
      contributors, 
      copyright, 
      id, 
      license,
      title
    };
```